config/app.php

```diff
@@ -64,9 +80,15 @@
 	| the Artisan command line tool. You should set this to the root of
 	| your application so that it is used when running Artisan tasks.
 	|
+	| url : the base url of your Lychee install up to the tld
+	| dir_url : the path of your Lychee install from the tld, MUST CONTAIN / if not left empty
```

Comment:
This is ambiguous. Does `dir_url` need to START with a `/` or END with it? As defined, it could be neither (`abc/def` contains `/`) but I think it's meant to say START?

Reply:
Actually, I can remove the MUST CONTAIN because I am taking care of it directly now.

Reply:
@kamil4  Fixed: [6b7f4bd](https://github.com/LycheeOrg/Lychee/pull/2150/commits/6b7f4bd5e06b46ce5a33798616b75bca52d21521)